### PR TITLE
Fix kubeone reset

### PR DESCRIPTION
**Pull request by kron4eg**
```release-note
Fix kubeone reset
```

### CODE REVIEW

File: pkg/templates/externalccm/ccm.go [hunk 1]
```diff
@@ -69,7 +69,7 @@ func waitForInitializedNodes(s *state.State) error {
 
 	s.Logger.Info("Waiting for nodes to initialize by CCM…")
 
-	return wait.Poll(5*time.Second, 60*time.Second, func() (bool, error) {
+	return wait.Poll(5*time.Second, 10*time.Minute, func() (bool, error) {
```
Comment (xmudrii) [inline pkg/templates/externalccm/ccm.go]: Why did you increase the timeout this much?

Comment (kron4eg) [reply pkg/templates/externalccm/ccm.go]: Sometimes cloud API is flaky (like in DO case, but could happen to anyone).

**Comment by xmudrii** [comment]
/hold
I want to discuss something.

**Pull request merged**